src/content/blog/astro-tutorial.mdx

```diff
@@ -0,0 +1,52 @@
+description: 'Wrapping up Astro Docs week with our biggest Docs announcement to date: our first Astro Tutorial.'
+publishDate: 'November 4, 2022'
+socialImage: '/assets/blog/astro-tutorial/cover.png'
+coverImage: '/assets/blog/astro-tutorial/cover.png'
+lang: 'en'
+authors:
+    - sarah
+    - chris
+    - dan
+    
+---
+
+import CTA from '/src/components/CallToAction.astro'
+import BlogContentImage from '/src/components/BlogContentImage.astro'
+import community from '/src/assets/blog/docs-hacktoberfest/contributors.jpg'
+
+We hope you enjoyed Astro Docs Week, where our Docs core maintainers took control of the keyboard and our Twitter account! Each day, we posted a peek behind the scenes of life on Team Docs:
+
+- our [low-code Hacktoberfest drive](https://twitter.com/astrodotbuild/status/1587160080717434881)
+- our [custom-built translation status tracker](https://twitter.com/astrodotbuild/status/1587547028258582528)
+- our ways of publicly recognizing contributors including [Astro Badges](https://twitter.com/astrodotbuild/status/1587901468585205760) and personalized [first-time contributor celebrations](https://twitter.com/astrodotbuild/status/1588285834054148096)
+
+To wrap up the week, we are excited to share the biggest Docs project to date: our first Astro tutorial.
+
+
+## Hello, world!
+
+We know that there's only so much you can read; sometimes, you need to build. In this first project-based tutorial, you will build all the basic pieces of an Astro blog from scratch.
+
+Astro, like the web, is for everyone. It was important to us to make this tutorial accessible to those just getting started with coding. You can find guidance on setting up your dev environment and using the terminal to get started. You'll even find instructions for completing the project entirely in the cloud. Pick your favorite online coding environment and go, even on a Chromebook or a tablet!
+
+So if you wanted to try Astro, but weren't sure where to start, check out our [Build a Blog tutorial](https://docs.astro.build/en/tutorial/0-introduction/) in Docs! And if you're a seasoned Astronaut, you now have one more place you can send your Astro-curious friends.
+
+<CTA primary href="https://docs.astro.build/en/tutorial/0-introduction/">Build a blog today!</CTA>
+
+## We’re just getting started…
+
+This tutorial is hot off the presses, and we’re looking for feedback on how to improve it. We also have new ideas for more advanced topics. Be sure to reach out to Team Docs on Discord to let us know what you'd like to see next!
+
+A huge thank you goes out to everyone involved in this challenging project! This required a lot of new UI for our Docs site, from interactive components to navigation. [Chris Swithinbank](https://twitter.com/swithinbank)
+ worked tirelessly to turn words into an experience. We also had some early feedback from community members. This, with [Dan Jutan](https://twitter.com/jutanium)'s relentless editing, helped focus and refine our direction.
```

Comment:
I could stand to see even more thanks to the community folks who joined us on calls/already PR’d the tutorial, etc. (Wording just a suggestion.)

```suggestion
 worked tirelessly to turn words into an experience. We also had some early feedback from community members. Thanks to all of those who’ve joined us on calls and contributed fixes over the past weeks. This, with [Dan Jutan](https://twitter.com/jutanium)'s relentless editing, helped focus and refine our direction.
```